scss/civicrm/administer/settings/_connection.scss

```diff
@@ -0,0 +1,8 @@
+.crm-connection-block {
+  @include civicrm-table;
+
+  h3 {
+    @include panel-header;
+    margin-bottom: 0;
+  }
+}
```

Comment:
set up your IDE so that it automatically leaves an empty line on save

Reply:
Done. and set IDE to do this automatic